docs/wip/application_health_probes.md

```diff
@@ -230,6 +230,92 @@ A listener for the new proxy-exposed TCP endpoint on port 15901 mapping to the c
 }
 ```
 
+### `tcpSocket`
```

Comment:
Is there an issue tracking promoting this wip doc into the website?

Reply:
No issue that I know of. I'll open up a follow-up PR that will move this doc and the control plane health probe doc. I don't think there's anything preventing those from promoting those.